frontend/src/App/App.tsx

```diff
@@ -27,6 +28,10 @@ function App() {
     await FirebaseAuth.signOut()
   }
 
+  const handleSearch = (results: any) => {
+    setSearchResult(results);
+  } 
```

Comment:
for one line callbacks like this, consider writing them 'in-line' in the component definition like '<Search searchCallBack={(results : any) => setSearchResults(results)}/>' ? IMO is a little cleaner, but just my opinion.